src/rust/zh.copymanga/res/settings.json

```
@@ -0,0 +1,24 @@
+[
+	{
+		"type": "group",
+		"title": "Settings",
+		"items": [
+			{
+				"type": "select",
+				"key": "imageQuality",
+				"title": "Image Quality",
+				"values": ["c800x", "c1200x", "c1500x"],
+				"titles": ["Low", "Medium", "High"],
+				"default": "c1500x"
+			},
+			{
+				"type": "select",
+				"key": "imageFormat",
+				"title": "Image Format",
+				"values": ["jpg", "webp"],
+				"titles": ["JPG", "WebP"],
+				"default": "jpg"
+			}
+		]
+	}
+]
```

should have a newline
```suggestion
]

```